match/spec/runner_spec.rb

```diff
@@ -223,6 +224,103 @@
           end.to raise_error("Your certificate 'E7P4EE896K.cer' is not valid, please check end date and renew it if necessary")
         end
 
+        it "renews an outdated certificate", requires_security: true do
```

Comment:
that's a huge test. I wonder how stable it will be. :)

I liked when you used GIVEN/WHEN/THEN in the other PR. If there was one method where I would add it, it's definitively this one.

Reply:
That's a tough one 😅. It's just an adapted copy-paste version from other tests. I'll try to make it look better.